### ISSUE TITLE
produced 7zip archive has broken header

### ISSUE THREAD

**Issue by miurahr** [miurahr/py7zr#34]
test case 'test_py7zr_write_single_close' produces a file named 'target.7z'. When running '7z l target.7z' output said its header has ERROR as follows:

```bash
$ 7z l pytest-current/test_py7zr_write_single_close0/target.7z 

7-Zip [64] 16.02 : Copyright (c) 1999-2016 Igor Pavlov : 2016-05-21
p7zip Version 16.02 (locale=ja_JP.UTF-8,Utf16=on,HugeFiles=on,64 bits,4 CPUs Intel(R) Core(TM) i3-6100U CPU @ 2.30GHz (406E3),ASM,AES-NI)

Scanning the drive for archives:
1 file, 120 bytes (1 KiB)

Listing archive: pytest-current/test_py7zr_write_single_close0/target.7z

--
Path = pytest-current/test_py7zr_write_single_close0/target.7z
Type = 7z
ERRORS:
Headers Error
Physical Size = 120
Headers Size = 83
Solid = -
Blocks = 0

   Date      Time    Attr         Size   Compressed  Name
------------------- ----- ------------ ------------  ------------------------
------------------- ----- ------------ ------------  ------------------------
                                     0            0  0 files

Errors: 1
```

**Comment by miurahr** [comment]
Work is progress on master branch.

**Comment by miurahr** [comment]
fixed and released v0.5a3
